src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -1152,5 +1435,52 @@ private static int LocateLastFoundByte(ulong match)
                                                        0x03ul << 32 |
                                                        0x02ul << 40 |
                                                        0x01ul << 48) + 1;
+
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static unsafe byte LoadByte(ref byte start, IntPtr offset)
```

Comment:
Is the disassembly exactly same with all these tiny helpers?

It is very easy to start running against the JIT inlining budged with many calls to tiny helpers like these.

Reply:
Yes, though can drop them, just looked a little cleaner

Reply:
Dropped `LoadByte`